conda_forge_tick/audit.py

```diff
@@ -16,7 +16,7 @@
 
 from conda_forge_tick.contexts import MigratorSessionContext, FeedstockContext
 from conda_forge_tick.git_utils import feedstock_url
-from conda_forge_tick.git_xonsh_utils import fetch_repo
+from conda_forge_tick.git_utils import fetch_repo
```

Comment:
Can you merge this import with the one above?

Reply:
What do you mean? I changed the imports everywhere in the code base.

Reply:
🤦  hold on

Reply:
done.